easybuild/easyconfigs/p/phono3py/phono3py-2.7.0-foss-2023a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonPackage'
+
+name = 'phono3py'
+version = '2.7.0'
+
+homepage = 'http://phonopy.github.io/phono3py/'
+description = """A simulation package of phonon-phonon interaction related properties."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://github.com/phonopy/phono3py/archive/refs/tags/']
```

Comment:
Use this instead
```
github_account = 'phonopy'
source_urls = [GITHUB_SOURCE]
```
And double check the checksum, the test build I just did complained about the checksum